spec/scenario/mock_project.rb

```diff
@@ -0,0 +1,48 @@
+module Mock
+  Foo = Struct.new(:foo_id) # model
```

Comment:
not important, but for those kind of examples, I prefer something less abstract, e.g. users, posts, etc. foos/bars are OK for unit tests but are hard to relate/follow when looked at in a doc

Reply:
Let's merge as is for now, I'll maybe rename them later.